src/planning/test/plan/planificator-test.ts

```diff
@@ -159,12 +159,14 @@ describe('remote planificator', () => {
       await verifyReplanningAndConsuming(1, ['Show products from your browsing context']);
 
       // Accept suggestion.
+      // eslint-disable-next-line require-atomic-updates
```

Comment:
This looks annoying. I understand what the rule is attempting to enforce, but here the race condition cannot happen. I would say disable this rule, as it's false positive rate would be too high. WDYT?

Reply:
Agreed.  I fixed this anyway by using a second variable.

Updated eslintrc.json.